lib/potassium/recipes/database_container.rb

```diff
@@ -28,11 +28,12 @@ class Recipes::DatabaseContainer < Rails::AppBuilder
     YAML
 
   def create
-    copy_file '../assets/docker-compose.yml', 'docker-compose.yml'
+    db_type = get(:database)
+    return if db_type == :none
```

Comment:
I think this should be as in the frontend recipe, as in some cases the value is `:None` and in others `:none`.

```ruby
return if [:None, :none].include? db_type.to_sym
```

Reply:
o `db_type.downcase.to_sym`

Reply:
@rjherrera that method `get()` is only used for getting results from the args and the questions form the "ui"??? if that is so, we could add some functionality to the `get` method to return nil whenever is `:None` or `:none`, something like that.

Reply:
I think so, I could test it but I remember having problems with recipes that didn't cover the `:None` case. Your suggestion seems to be a nice solution!